build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
```

Comment:
The property FloatingTestRuntimeDependencies was originally designed for testing the latest dependencies by Corefx team.  The design has been reviewed by both Corefx and WCF team, and implemented by Corefx. The implementation did not work as expected and I figured out it's because TestFixedRuntimeProjectJson cannot be configured. This PR fixes it.